trlx/trainer/accelerate_ilql_trainer.py

```diff
@@ -95,3 +142,73 @@ def save_pretrained(self, directory: Optional[str] = None):
             "`AccelerateILQLTrainer` does not currently support automatic saving "
             "with `transformers.PreTrainedModel.save_pretrained`."
         )
+
+    def make_experience(self, samples, rewards, max_length=2048):
```

Comment:
maybe pull this into somewhere it can be shared with nemo impl?

Reply:
I've duplicated the `make_experience` into NeMo for now. There was a subtle difference in logging whereby NeMo couldn't recognize the global rank `RANK == 0`checks forcing each rank to write tables to stdout (the fix is to just use their global rank check util).

I think it might be best to push this off to another PR because we'll need to re-visit this abstraction again for PPO. What do you think?

Reply:
Yeah, I think that makes sense for now. Maybe `torch.distributed.get_rank()` will work for both but we can revisit